### PR TITLE
Extend docs

### PR DESCRIPTION
**Pull request by gabrielfs7**
Extend docs:

https://oat-sa.atlassian.net/browse/ADF-8

PRS related:

- [ ] https://github.com/oat-sa/tao-core/pull/2874
- [ ] https://github.com/oat-sa/extension-tao-mediamanager/pull/264/files
- [ ] https://github.com/oat-sa/extension-tao-dac-simple/pull/162

### CODE REVIEW

**Review by mike-ionut-mihai-sandu-tao — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

File: README.md [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
 
-Simple Data Access Control allows the restriction of which user can access which resources, in the way compatible with Advanced Search.
+Simple Data Access Control allows the restriction of which user can access which resources, in the way compatible with
+Advanced Search.
 
 Access Privileges are granted either to users directly or to roles, applying to all users who have that specific role.
 
-Privileges are given per resource, so that in order to remove the write access to all items within a class, the new access rights need to be applied recursively to all resources by checking "recursive" before saving the changes.
+Privileges are given per resource, so that in order to remove the write access to all items within a class, the new
+access rights need to be applied recursively to all resources by checking "recursive" before saving the changes.
```
Comment (ktsirkou) [inline README.md:10]: We should explain the 2 "save strategies" we have as part of `taoDacSimple/PermissionsService.conf.php` 
```
<?php
/**
 * Default config header created during install
 */

return new oat\taoDacSimple\model\PermissionsServiceFactory(array(
    'save_strategy' => 'oat\\taoDacSimple\\model\\SyncPermissionsStrategy',
    'recursive_by_default' => false
));


```
- `SyncPermissionsStrategy` (Default) 
When saving -> overwrite what permissions existed with new ones as given from user.
- `SavePermissionsStrategy`
When saving -> merge existing permission with changes (permission addition,removals) from the user

It's worth to emphasize that "recursive" save, especially with `SyncPermissionsStrategy` is very dangerous.

**Review by boajer — APPROVED** [review]

**Review by shpran — APPROVED** [review]
LGTM 👍🏻

File: README.md (hunk 1 above)
Comment (gabrielfs7) [reply README.md:10]: Ok, added to the doc. Thanks for the feedback

**Review by ktsirkou — APPROVED** [review]
Looks good to me!

**Pull request merged**